api_files/dependencies.json

```diff
@@ -79,5 +79,12 @@
                 }
             }
         ]
+    },
+    "fromList": {
```

Comment:
It's too bad we can't add something for `state.mission.critical_ids`.  But it's the keys of those entries we need to validate against `state.characters[].id`, so it doesn't work.

Reply:
Yeah, I just had to do that separately because it's such a weird case. It's in another PR